mapper_visitor/src/visitor.rs

```diff
@@ -0,0 +1,16 @@
+use chrono::{DateTime, FixedOffset};
+
+pub trait MeasurementVisitor {
+    type Error;
+
+    fn visit_measurement_type(&mut self, typename: &str) -> Result<(), Self::Error>;
+    fn visit_timestamp(&mut self, timestamp: DateTime<FixedOffset>) -> Result<(), Self::Error>;
```

Comment:
Why a `FixedOffset` and not `Utc`?

Reply:
I feel, its to preserve the time zone that is provided by the user. If we use FixedOffset, we can use Utc or Local. Else we need to convert the local time to Utc always.